### ISSUE TITLE
Check CLI arguments like --output before executing evals

### ISSUE THREAD

**Issue by efung** [promptfoo/promptfoo#1129]
The CLI argument `--output` expects a filename with a supported extension to know how to write the output. However, it seems the argument is only checked when writing the output, _after_ the evals have been performed.

The arguments should be checked _before_ the (expensive) evals are run.

For example:

```sh
$ npx promptfoo@0.65.2 eval --config promptfooconfig.yaml --output eric/test-one --no-table --filter-first-n 1
[████████████████████████████████████████] 100% | ETA: 0s | 1/1 | Sample full configuration "{{prompt}}" …
Error: Unsupported output file format eric/test-one, please use csv, txt, json, yaml, yml, html.
    at writeOutput (/Users/efung/src/eval/node_modules/promptfoo/dist/src/util.js:409:19)
```

**Comment by mldangelo** [comment]
Addressed in https://github.com/promptfoo/promptfoo/pull/1135

**Comment by mldangelo** [comment]
Hi @efung, Thank you for highlighting this issue! We've just released an update that addresses it. The CLI now validates the `--output` file extension before running evaluations, as suggested. This fix is included in [0.70.1](https://github.com/promptfoo/promptfoo/releases/tag/0.70.1) Thanks again for your suggestion!
